### PR TITLE
#44: Add logging (closes #44)

### LINKED ISSUE #44: Add logging
## requirements
- mailo should log operations

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by calippo**
Closes #44

### tests performed
- logging is working in tests

### CODE REVIEW

File: src/main/scala/mailo/package.scala [hunk 1]
```diff
@@ -87,6 +91,7 @@ class Mailo(
         attachments = attachments,
         tags = tags
       ))
+      _ = logger.info(s"email sent with result: $result")
```
Comment (bytecodeguru) [inline src/main/scala/mailo/package.scala]: Better to stick with `.debug`, imho.

**Review by bytecodeguru — DISMISSED** [review]
LGTM 👍 
just a fix for logging level level

File: src/main/scala/mailo/package.scala (hunk 1 above)
Comment (calippo) [reply src/main/scala/mailo/package.scala]: I deliberately decided to stick with `info`. This is unlikely going to fill the disk with annoying info (if that is the case, it means we are spamming too many emails :p).
I think it's a useful information, if you really want you can anyhow filter it out later with log4j or logback.

The idea was to allow the user to log one line per email sent. Debug level currently contains too much stuff.

**Review by bytecodeguru — APPROVED** [review]
💯

**Pull request merged**